Dockerfile

```diff
@@ -1,17 +1,27 @@
-FROM node:14-slim
+FROM node:14-slim as build
 
 LABEL maintainer="Orta Therox"
 LABEL "com.github.actions.name"="Danger JS Action"
 LABEL "com.github.actions.description"="Runs JavaScript/TypeScript Dangerfiles"
 LABEL "com.github.actions.icon"="zap"
 LABEL "com.github.actions.color"="blue"
 
-RUN mkdir -p /usr/src/danger
-COPY . /usr/src/danger
-RUN cd /usr/src/danger && \
-  yarn && \
-  yarn run build:fast && \
-  chmod +x distribution/commands/danger.js && \
-  ln -s $(pwd)/distribution/commands/danger.js /usr/bin/danger
+WORKDIR /usr/src/danger
+RUN yarn global add yarn-audit-fix
+COPY package.json yarn.lock ./
+RUN yarn install && \
+    yarn-audit-fix
+COPY . .
+RUN yarn run build:fast
+RUN rm -rf node_modules
```

Comment:
don't `rm -rf node_modules`, yarn removes superfuous packages (from dev dependencies) already itself

Reply:
I haven't touched this for a bit, just looking over your reviews now. But IIRC it wasn't removing them... or it was applying some sort of heuristic that was resulting in some build-time dependencies not being removed. FWIW I'm not really a node/js guy; but wanted to use this tool and it SEEMED like the container was much larger than it needed to be

Reply:
https://github.com/wagoodman/dive might be useful to check what's in the layers